### PR TITLE
Create assets for all branches and tags

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

File: src/rewrite_app_repos.sh [hunk 1]
```diff
@@ -52,13 +52,11 @@ do
...
       json_path=".require.\"greenpeace/${reponame}\""
-      plugin_version=$(jq -r "${json_path}" < "${f}")
+      plugin_version=$(jq -r "${json_path} // empty" < "${f}")
       branch=${plugin_version#"$composer_dev_prefix"}
 
-      # if these are the same do nothing as plugin is not in file or it doesn't start with "dev-"
-      # master already includes built files
-      if [[ "${branch}" != "${plugin_version}" ]] && [ "${branch}" != master ]; then
-        # dev branches do not include the built assets, only master does. So we need to build them here.
+      if [ -n "$branch" ]; then
```
Comment (Inwerpsel) [inline src/rewrite_app_repos.sh]: It probably also needs to build if the plugin is required at a specific version instead of through `master` branch?

Comment (comzeradd) [reply src/rewrite_app_repos.sh]: `$branch` actually is set with a tag also, when a tag is used in a composer file. So it has the proper value, without a `dev-` prefix when a tag is used.

This also works with `git clone`, that's a few lines below. eg.
```
git clone --recurse-submodules --single-branch --branch v1.78.2 https://github.com/greenpeace/planet4-master-theme
```

Comment (comzeradd) [reply src/rewrite_app_repos.sh]: Here is an example:
https://circleci.com/gh/greenpeace/planet4-test-tavros/55

Comment (Inwerpsel) [reply src/rewrite_app_repos.sh]: Ah yeah, I missed that `"${branch}" != "${plugin_version}"` was the check that was ensuring that only `dev-` versions are built, but that has been removed, so you can ignore my comment.

**Pull request merged**